sdk/boms/spring-cloud-azure-dependencies/pom.xml

```diff
@@ -128,6 +128,31 @@
         <artifactId>spring-cloud-azure-starter-appconfiguration</artifactId>
         <version>${project.version}</version>
       </dependency>
+      <dependency>
+        <groupId>com.azure.spring</groupId>
+        <artifactId>spring-cloud-azure-starter-appconfiguration-config</artifactId>
+        <version>${project.version}</version>
+      </dependency>
+      <dependency>
+        <groupId>com.azure.spring</groupId>
+        <artifactId>spring-cloud-azure-appconfiguration-config</artifactId>
+        <version>${project.version}</version>
+      </dependency>
+      <dependency>
+        <groupId>com.azure.spring</groupId>
+        <artifactId>spring-cloud-azure-appconfiguration-config-web</artifactId>
+        <version>${project.version}</version>
+      </dependency>
+      <dependency>
+        <groupId>com.azure.spring</groupId>
+        <artifactId>spring-cloud-azure-feature-management</artifactId>
+        <version>${project.version}</version>
+      </dependency>
+      <dependency>
+        <groupId>com.azure.spring</groupId>
+        <artifactId>spring-cloud-azure-feature-management-web</artifactId>
+        <version>${project.version}</version>
+      </dependency>
```

Comment:
Please check wether this has been in the main branch.

Reply:
Sure, already exist https://github.com/Azure/azure-sdk-for-java/blob/6bae1eea54eef12abc37704965fdc5cad69ed5f9/sdk/boms/spring-cloud-azure-dependencies/pom.xml#L127